scoring.js

```diff
@@ -0,0 +1,53 @@
+const calculateScore = (player) => {
+    switch (player.position) {
+        case 'QB':
+            return passingPoints(player) + rushingPoints(player)
+
+        case 'WR':
+
+        case 'RB':
+            return rushingPoints(player) + receivingPoints(player) + kickScore(player)
+
+        case 'TE':
+            return receivingPoints(player)
+
+
+
+        default:
+            return 0
+    }
+}
+const passingPoints = (player) => {
+    const yrd = player.stats.passing.yards / 25
+    const td = player.stats.passing.touchdowns * 6
+    const int = player.stats.passing.interceptions * -3
+
+    return yrd + td + int
+}
+const rushingPoints = (player) => {
```

Comment:
Try to clean up your spacing. It's minor but stands out to senior devs and marks you as junior when someone sees it. Always have blank lines between function definitions. Also you don't need that extra space in the switch since the natural structure of the switch makes it readable. Sometimes the extra space can help while working but always do a last sweep through and clean up stuff like that.